lib/fusion_auth/test_utilities.ex

```diff
@@ -0,0 +1,151 @@
+defmodule FusionAuth.TestUtilities do
+  alias FusionAuth.Users
+  alias FusionAuth.Applications
+  alias FusionAuth.Registrations
+  alias FusionAuth.OpenIdConnect
+
+  defp rand_string() do
```

Comment:
So instead of a custom function for a random string, we use the [faker](https://github.com/elixirs/faker) library with our other applications, just FYI